modules/snooker/src/main/ts/ephox/snooker/api/TableDirection.ts

```diff
@@ -3,26 +3,20 @@ import { Option } from '@ephox/katamari';
 import { Element } from '@ephox/sugar';
 import { BarPositions, ColInfo } from '../resize/BarPositions';
 
-export default function (directionAt: (e: Element) => { isRtl: () => boolean }): BarPositions<ColInfo> {
-  const auto = function (table: Element) {
-    return directionAt(table).isRtl() ? ResizeDirection.rtl : ResizeDirection.ltr;
-  };
+export type TableDirection = BarPositions<ColInfo>;
 
-  const delta = function (amount: number, table: Element) {
-    return auto(table).delta(amount, table);
-  };
+export default function (directionAt: (e: Element) => { isRtl: () => boolean }): TableDirection {
```

Comment:
This would be better if you didn't `export default` here and instead did `export const TableDirection = `, as that way the types would merge and you wouldn't have to do `TableDirection.TableDirection` which is a little weird.

Reply:
Actually looking at this more, with the current changes and the import/export in `Main.ts`, the export above does nothing as the type still will not be exported via `Main.ts`. That means these changes doesn't solve the original issue.

Reply:
I overlooked this so thanks for picking that up. This should be fixed now.